common/src/main/java/com/wynntils/functions/generic/StringFunctions.java

```diff
@@ -43,6 +44,33 @@ public FunctionArguments.RequiredArgumentBuilder getRequiredArgumentsBuilder() {
         }
     }
 
+    public static class FormatRangedFunction extends GenericFunction<String> {
+        @Override
+        public String getValue(FunctionArguments arguments) {
+            RangedValue value = arguments.getArgument("value").getRangedValue();
+            return StringUtils.integerToShortString(value.low()) + "/" + StringUtils.integerToShortString(value.high());
```

Comment:
We represent RangedValues as `<low>-<high>` in our toString method. I think we should do the same here. That is kind of the essential difference between a ranged value ("between low and high") and a capped value `<cur>/<max>` ("the value cur out of max").

Reply:
```suggestion
            return StringUtils.integerToShortString(value.low()) + "-" + StringUtils.integerToShortString(value.high());
```

Indeed! I just copied the method, and forgot to change it. Thanks!